.github/workflows/e2e-tests-main-devnet.yml

```diff
@@ -448,6 +448,11 @@ jobs:
           chmod +x target/release/aleph-node
           docker build --tag aleph-node:latest -f ./docker/Dockerfile .
 
+      - name: Build docker image for onfinality
+        id: build-image-onfinality
+        run: |
+          docker build --tag aleph-node:latest-onfinality -f ./docker/Dockerfile.onfinality .
+
```

Comment:
Maybe to speedup a build process I would add to this step `--cache-from` option from latest image. Example:
`docker build --cache-from aleph-node:latest-onfinality --tag aleph-node:latest-onfinality -f ./docker/Dockerfile.onfinality .`

Reply:
Added